docs/flink-getting-started.md

```diff
@@ -110,7 +110,7 @@ wget ${FLINK_CONNECTOR_URL}/${FLINK_CONNECTOR_PACKAGE}-${HIVE_VERSION}_${SCALA_V
 ## Flink's Python API
 
 {{< hint info >}}
-PyFlink 1.6.1 [does not work on OSX with a M1 cpu](https://issues.apache.org/jira/browse/FLINK-28786)
+PyFlink 1.16.1 [does not work on OSX with a M1 cpu](https://issues.apache.org/jira/browse/FLINK-28786)
```

Comment:
If it Flink version is 1.17.0, I think it can delete it.
https://flink.apache.org/2023/03/23/announcing-the-release-of-apache-flink-1.17/#pyflink